content/docs/for-app-developers/how-to/build-inputs/specify-default-launch-process.md

```diff
@@ -4,6 +4,19 @@ weight=99
 summary="Buildpacks can define multiple processes for an application image. Specify which process should be the default."
 +++
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+Buildpacks usually define the default process type for an application, while retaining the ability for a user to specify their desired default process.
 
-If you are familiar with this content and would like to make a contribution, please feel free to open a PR :)
+To configure the `build` to work differently from the default behavior:
+
+* You first need to know what processes would be contributed by the buildpacks running in your build.
+* Once known, you can append the following flag to the `pack build` command
+
+```bash
+pack build --default-process <process name> <image name>` # <process name> must be a valid process name in launch.toml
+```
+
+If this flag is not provided by the user, `pack` will provide the process type as `web` to the `lifecycle`.
```

Comment:
```suggestion
```

I think this is actually not true for newer Platform API versions. We should probably update the pack help text but that is another story